assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { dataFontEm } from '../../../../units';
+
+const DEFAULT_PRESET = {
```

Comment:
The issue mentions  default values for text alignment, box size, line height, etc.

Those seem to be all missing from this PR?

Reply:
So the box size will be in a different issue as discussed.

However text alignment is actually left aligned in figma (even though the issue mistakenly says center aligned). Since thats the default alignment, I don't think we need to add that.

I added line height in a new commit.